lib/models/command.js

```diff
@@ -197,7 +198,7 @@ let Command = CoreObject.extend({
       return this._currentTask.onInterrupt();
     } else {
       // interrupt all external commands which don't use `runTask()` with an error
-      process.exit(1);
+      return RSVP.reject();
```

Comment:
We care about exiting non-zero here.  Rejecting a promise on command does not affect the exit code.  This is why the tests are red for this PR as well.

Were you planning on making other changes to ensure that the process still exits non-zero and if so what would those changes be?